v2/account_claims.go

```diff
@@ -26,26 +26,45 @@ import (
 // NoLimit is used to indicate a limit field is unlimited in value.
 const NoLimit = -1
 
-// OperatorLimits are used to limit access by an account
-type OperatorLimits struct {
-	Subs            int64 `json:"subs,omitempty"`      // Max number of subscriptions
-	Conn            int64 `json:"conn,omitempty"`      // Max number of active connections
-	LeafNodeConn    int64 `json:"leaf,omitempty"`      // Max number of active leaf node connections
+type AccountLimits struct {
 	Imports         int64 `json:"imports,omitempty"`   // Max number of imports
 	Exports         int64 `json:"exports,omitempty"`   // Max number of exports
-	Data            int64 `json:"data,omitempty"`      // Max number of bytes
-	Payload         int64 `json:"payload,omitempty"`   // Max message payload
 	WildcardExports bool  `json:"wildcards,omitempty"` // Are wildcards allowed in exports
 }
 
+type NatsLimits struct {
+	Subs         int64 `json:"subs,omitempty"`    // Max number of subscriptions
+	Conn         int64 `json:"conn,omitempty"`    // Max number of active connections
+	LeafNodeConn int64 `json:"leaf,omitempty"`    // Max number of active leaf node connections
+	Data         int64 `json:"data,omitempty"`    // Max number of bytes
+	Payload      int64 `json:"payload,omitempty"` // Max message payload
+}
+
+type JetStreamLimits struct {
+	MemoryStorage int64 `json:"mem_storage,omitempty"`  // Max number of bytes stored in memory across all streams. (0 means disabled)
+	DiskStorage   int64 `json:"disk_storage,omitempty"` // Max number of bytes stored on disk across all streams. (0 means disabled)
```

Comment:
I don't think JetStream is able to have different storage policies at the same time? - If this is not the case and the storage is either `Memory` or `Disk`, likely we need to have storage enums, to specify the kind of storage and have a `Data` field that specifies the limit. This would match `NatsLimits`

Reply:
both are active - you can have a stream be memory and another be disk at the same time